docs/excel/custom-functions-best-practices.md

```diff
@@ -33,9 +33,12 @@ function getComment(x) {
 
 ## Troubleshooting
 
-If you are testing your add-in in Office on Windows, you should enable **[runtime logging](../testing/troubleshoot-manifest.md#use-runtime-logging-to-debug-your-add-in)** to troubleshoot issues with your add-in's XML manifest file, as well as several installation and runtime conditions. Runtime logging writes `console.log` statements to a log file to help you uncover issues.
+1. If you are testing your add-in in Office on Windows, you should enable **[runtime logging](../testing/troubleshoot-manifest.md#use-runtime-logging-to-debug-your-add-in)** to troubleshoot issues with your add-in's XML manifest file, as well as several installation and runtime conditions. Runtime logging writes `console.log` statements to a log file to help you uncover issues.
+
+2. Your add-in will not be installed if one or more custom functions conflicts with a previously registered add-in's custom functions. In this case, you can either remove the existing add-in, or if you encounter this error while developing an add-in, you can specify a different namespace name in your manifest.
```

Comment:
Is "install" the right term, or "load"?
Also is it possible to remove an add-in from a document? How do you do that?

Reply:
load works better. I don't know do we have a document that explains how to uninstall add-ins? I couldn't find